README.md

```diff
@@ -208,148 +233,52 @@ The number of object that should be indexed. Only used by the [`checker`](#check
 
 Default is `0`.
 
-## Test the configuration with the generator
+### Possible issues
 
-The generator allows to both generate the selectors and visualize on pages
-what is actually selected. Refer to `html/generator/README.md` to use it.
+#### Duplicated content
 
-## Test the UX/UI with the playground
+It could happen that the crawled website returned duplicated data. Most of the time, this is because the crawled pages got the same urls with two different scheme.
 
-To test it live, you can use the following HTML page:
+If there are `http://website.com/page` and `http://website.com/page/` (notice the second one ending with `/`), the scrapper will consider it as two different urls. This can be fixed adding a regex to the `stop_urls` in the `config.json`:
```

Comment:
If we have URLs like `http://website.com/page` and `http://website.com/page/` (notice the second one ending with `/`), the scrapper will consider them as different. This can be fixed by adding a regex to the `stop_urls` in the `config.json`: